models/stablelm/stablelm-jp-1b-rp_then_jav1-294b/result.json

```
@@ -0,0 +1,48 @@
+{
+  "results": {
+    "jnli-1.1-0.2": {
+      "acc": 0.4009860312243221,
+      "acc_stderr": 0.009936010515698028,
+      "acc_norm": 0.3635990139687757,
+      "acc_norm_stderr": 0.009752269983725555
+    },
+    "marc_ja-1.1-0.2": {
+      "acc": 0.7771189396581792,
+      "acc_stderr": 0.005496539565709208,
+      "acc_norm": 0.7771189396581792,
+      "acc_norm_stderr": 0.005496539565709208
+    },
+    "jcommonsenseqa-1.1-0.2": {
+      "acc": 0.2680965147453083,
+      "acc_stderr": 0.013248038756079302,
+      "acc_norm": 0.24039320822162646,
+      "acc_norm_stderr": 0.012780110667692907
+    },
+    "jsquad-1.1-0.2": {
+      "exact_match": 56.39351643403872,
+      "f1": 66.59905762410258
+    }
+  },
+  "versions": {
+    "jnli-1.1-0.2": 1.1,
+    "marc_ja-1.1-0.2": 1.1,
+    "jsquad-1.1-0.2": 1.1,
+    "jcommonsenseqa-1.1-0.2": 1.1
+  },
+  "config": {
+    "model": "hf-causal",
+    "model_args": "pretrained=/fsx/proj-jp-stablegpt/hf_model/1b-rp_then_jav1-294b,tokenizer=/fsx/proj-jp-stablegpt/tokenizers/nai-hf-tokenizer/,use_fast=False",
```

@iwiwi maybe we should hide the full path as follows. 
```suggestion
    "model_args": "pretrained=${PROJECT_DIR}/hf_model/1b-rp_then_jav1-294b,tokenizer=${PROJECT_DIR}/tokenizers/nai-hf-tokenizer/,use_fast=False",
```

reference: https://github.com/Stability-AI/lm-evaluation-harness/blob/jp-stable/models/stablelm/stablelm-jp-instruct-3b_1.3.0/result.json#L34